sos/report/plugins/coredump.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2023 Red Hat, Inc., Jose Castillo <jcastillo@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin, PluginOpt
+
+
+class Coredump(Plugin, RedHatPlugin):
+
+    short_desc = 'Retrieve core dumps information'
+
+    plugin_name = "coredump"
+    profiles = ('system', 'debug')
+    packages = ('systemd-udev',)
+
+    option_list = [
+        PluginOpt("detailed", default=False,
+                  desc="collect detailed information for every report")
+    ]
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/systemd/coredump.conf",
+            "/etc/systemd/coredump.conf.d/",
+            "/run/systemd/coredump.conf.d/",
+            "/usr/lib/systemd/coredump.conf.d/"
+        ])
+
+        self.add_cmd_output("coredumpctl dump")
+
+        coredump_list = self.collect_cmd_output("coredumpctl list")
+        if self.get_option("detailed") and coredump_list['status'] == 0:
+            for line in coredump_list["output"].splitlines():
+                if not line.startswith("TIME"):
```

Comment:
is there a meaningful else to this "if"? What if it does start with "TIME" but it might still be useful getting stuff like info on whether the coredump can be provided for more analysis if needed, i.e if its still around?

           COREFILE
               Information whether the coredump was stored, and whether it is still accessible: "none" means the core was not stored, "-" means that it
               was not available (for example because the process was not terminated by a signal), "present" means that the core file is accessible by the
               current user, "journal" means that the core was stored in the "journal", "truncated" is the same as one of the previous two, but the core
               was too large and was not stored in its entirety, "error" means that the core file cannot be accessed, most likely because of insufficient
               permissions, and "missing" means that the core was stored in a file, but this file has since been removed.